### PR TITLE
read me file

### PR DESCRIPTION
**Pull request by donia98**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,48 @@
 # voyage
-App to collect all entertainments places at Gaza
 
-### DataBase : 
+### What is voyage ?!
+A web application that allows you to search and explore places of entertainment in Gaza and gives you the possibility to give your opinion in places after you register as a regular user on the site ..
```
Comment (FarahZaqout) [inline README.md]: Grammar correction: Give your opinion on places

File: README.md [hunk 2]
```diff
@@ -1,8 +1,48 @@
...
+If you are the owner of an entertainment site and want to market it you can register as a business account and then you can view your services and receive views of people about the services and the place.
```
Comment (FarahZaqout) [inline README.md]: Receive reviews. (instead of receive views)

File: README.md [hunk 3]
```diff
@@ -1,8 +1,48 @@
...
+
+
+### Why?!
+People in Gaza often suffer from difficult access to suitable recreational places, for example a family that wants to go to a park and looking for the nearest one that is clean and quiet. Our site will allow them to search for parks in Gaza and nearby
```
Comment (FarahZaqout) [inline README.md]: our site will allow them to search for nearby parks in Gaza

File: README.md [hunk 4]
```diff
@@ -1,8 +1,48 @@
...
+It will also support and market recreational venues.
+
+### User Stories :
+
+ **As a person:**
+  - I have a profile page.
+  - I can view all places in Gaza
+  - I can add a review of entertainment places.
+  - I can add a rate this place.
```
Comment (FarahZaqout) [inline README.md]: I can rate this place.

**Review by FarahZaqout — CHANGES_REQUESTED** [review]
Few grammar errors.

**Comment by donia98** [comment]
I resolved the grammar error

**Pull request merged**